### PR TITLE
Weekly 10

### PR DESCRIPTION
**Pull request by RichardLitt**
Stubbed. May be merged into 9.

### CODE REVIEW

File: published/010-2016-April-18.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [April 6th through April 18th](//github.com/ipfs/pm/issues/101) sprint:
+
+## Updates
+
+### 0.4.0
+
+...
```
Comment (jbenet) [inline published/010-2016-April-18.md]: short description, and link to the 0.4.0 blogpost for more.

File: published/010-2016-April-18.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+@JGAntunes, @dignifiedquire, @nginnever, @diasdavid, and @noffle have been working hard to bring [the pantheon of js-ipfs modules](https://github.com/ipfs/js-ipfs/issues?utf8=%E2%9C%93&q=is%3Aissue+coverage+dignified) up to snuff: detailed READMEs with examples and full API docs, as well as 100% test coverage. We made significant inroads last sprint on [js-peer-id](https://github.com/diasdavid/js-peer-id), [js-multiaddr](https://github.com/jbenet/js-multiaddr), [js-peer-info](https://github.com/diasdavid/js-peer-info), [js-ipfs-blocks](https://github.com/ipfs/js-ipfs-blocks/) and more! There are a lot more modules that still need good READMEs and docs, so [dive in](https://github.com/ipfs/js-ipfs/issues?utf8=%E2%9C%93&q=is%3Aissue+coverage+dignified+is%3Aopen) and give us a hand!
+
+## Community
+
+### Peergos
+
+Work on [Peergos](https://github.com/ianopolous/Peergos) is progressing well. We're close to getting a FUSE binding working. Throughput hasn't been as bad as we were expecting with all the encryption and erasure coding, and it's looking likely that it will end up being network IO bound, instead of CPU bound for non trivial writes.
```
Comment (jbenet) [inline published/010-2016-April-18.md]: change to third person, as this is ianopolous's project, not the core ipfs team's

File: published/010-2016-April-18.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Community
+
+### Peergos
+
+Work on [Peergos](https://github.com/ianopolous/Peergos) is progressing well. We're close to getting a FUSE binding working. Throughput hasn't been as bad as we were expecting with all the encryption and erasure coding, and it's looking likely that it will end up being network IO bound, instead of CPU bound for non trivial writes. 
+
+### SpaceX Landing
+
+Things are awesome and getting awesomer.
```
Comment (jbenet) [inline published/010-2016-April-18.md]: haha yes pls

**Comment by jbenet** [comment]
Lots of great stuff to mine in https://github.com/ipfs/pm/issues/101

**Comment by jbenet** [comment]
In particular would love to see a big update drop on Orbit here

**Comment by ghost** [comment]
Is this the issue for the tomorrow/wednesday/next weekly?

### Content-Type of The Week

This week's Content-Type is the non-standard `chemical/x-chemdraw`. ChemDraw is a molecule editor first released in 1985, and still maintained as of today. We don't know which molecules were added to IPFS, but will do our best for IPFS to be around in 30 years just like ChemDraw is still around.

Every week we highlight a Content-Type that we've seen on the public HTTP-to-IPFS gateway at https://ipfs.io. If you're waiting to see one particularly interesting Content-Type in one of the next Weeklies, propose it by `ipfs add -w`'ing a file of that type, and requesting it from the public gateway: https://ipfs.io/ipfs/<hash>

**Comment by RichardLitt** [comment]
Should be ready to go, @jbenet. Needs the contributors, still - having an issue with name-your-contributors. Looking into it.

**Comment by eminence** [comment]
@RichardLitt The only code contributor that was not included in your list is:

FrauBienenstich https://github.com/ipfs/js-ipfsd-ctl/commits/master?author=FrauBienenstich

**Comment by RichardLitt** [comment]
Added! Thanks!

**Comment by jbenet** [comment]
Reviewed. this content is good.

Missing:
- I spoke at the Berkman Center, there's pictures on twitter. Something like:
  
  > @jbenet and @nicolagreco spoke about IPFS with fellows at the Berkman Center. They discussed the core motivations of the IPFS Project, how it works, the state of the implementations, and a wide array of issues IPFS tackles. The discussion was excellent, starting with important ethical problems present in the web today, going through critical features our web should exhibit to solve them, and ending with how to get there. We look forward to tighter collaboration between our communities. Thank you very much for having us!
- We had the IPFS meetup in boston

And, the writeup on the NYC meetup should land in the blog too.

**Pull request merged**